src/manifolds/FlagStiefel.jl

```diff
@@ -0,0 +1,179 @@
+
+"""
+    default_inverse_retraction_method(M::Flag)
+
+Return [`PolarInverseRetraction`](https://juliamanifolds.github.io/ManifoldsBase.jl/stable/retractions.html#ManifoldsBase.PolarInverseRetraction)
+as the default inverse retraction for the [`Flag`](@ref) manifold.
+"""
+default_inverse_retraction_method(::Flag) = PolarInverseRetraction()
+
+"""
+    default_retraction_method(M::Flag)
+
+Return [`PolarRetraction`](https://juliamanifolds.github.io/ManifoldsBase.jl/stable/retractions.html#ManifoldsBase.PolarRetraction)
+as the default retraction for the [`Flag`](@ref) manifold.
+"""
+default_retraction_method(::Flag) = PolarRetraction()
+
+"""
+    default_vector_transport_method(M::Flag)
+
+Return the [`ProjectionTransport`](https://juliamanifolds.github.io/ManifoldsBase.jl/stable/vector_transports.html#ManifoldsBase.ProjectionTransport)
+as the default vector transport method for the [`Flag`](@ref) manifold.
+"""
+default_vector_transport_method(::Flag) = ProjectionTransport()
+
+function inner(
```

Comment:
So opposed to what the docs say, Stiefel is our default ;) ?

Reply:
Yes, sorry for the confusion.